source/driver-read-preferences.rst

```diff
@@ -272,7 +272,7 @@ preference, but it only obeys the preference for these commands:
 
 * group
 * inline mapreduce
-* aggregate without $out specified
+* aggregate without write stage specified (e.g. $out)
```

Comment:
Suggestion: `(e.g. ``$out``, ``$merge``)`

Reply:
I originally didn't want to mention `$merge` in this document, since it's quite old (2014) and deprecated, but there's no harm in this while I'm revising the text to be more future-proof.